site/app/templates/EditPronounsForm.twig

```diff
@@ -29,6 +29,14 @@
             </input>
             <button aria-label="Clear pronoun input" type="button" style="background: none;border:none;" onclick="clearPronounsBox();"><i class="fas fa-trash" style="padding:0.5rem;" title="Clear pronouns field"></i></button>
         </div>
+        <br>
```

Comment:
I'm pretty sure we shouldn't use `<br>` but rather use CSS. Either way, the gap is too large so it looks a bit odd
![image](https://github.com/Submitty/Submitty/assets/47994802/580d3d11-cf9b-44c2-ba64-9895b2ecf530)

The gap should only be a little bit. But honestly, if you think it's unnecessary then we can just do without the gap and leave it to how it was before